artifacts/definitions/Windows/Forensics/CertUtil.yaml

```diff
@@ -93,6 +93,7 @@ sources:
                if(condition=AlsoUpload, then=upload(file=OSPath, accessor=Accessor)) AS _MetdataUpload,
                if(condition=AlsoUpload, then=upload(file=_ContentPath, accessor=Accessor)) AS _Upload,
                Header.URL AS URL,
+               parse_string_with_regex(string=Header.URL,regex='^(?:https?:\/\/)?(?:[^@\/\n]+@)?(?:www\.)?([^:\/\n]+)').g1 AS UrlTLD,
```

Comment:
is it better to use the url() function to parse this properly?

Reply:
It might be if it's simpler than the regex, I'll test on Monday 😊

Reply:
@scudette - Definitley simpler and works better, just updated now